.github/workflows/test.yaml

```diff
@@ -27,6 +27,13 @@ jobs:
       run: |
         export GOPATH=$(go env GOPATH)
         make verify-codegen
+    - name: Setup kustomize
+      uses: imranismail/setup-kustomize@v1
+      with:
+        kustomize-version: "3.1.0"
+    - name: Verify manifest consistency
```

Comment:
Looks like this check is failing at introduction 😁

Reply:
Ah, should have read _Special notes for your reviewer_ first. 🤦 

> How do we want to handle that?

I recommend a TDD-like here:
1. introduce a failing test
2. fix the cause of failure
3. anticipate test success :tada:

For me, this can (or even should) happen in 1 PR. It does not feel right to deliberately introduce a CI failure. I won't insist, though.

Reply:
Yeah, mostly a question of just where that change goes, which is a minor procedural point and not particularly important.

Given that it's pretty minor and I have the link to an existing test run demonstrating the failure already, going ahead and pushing the manifest updates within this PR.